src/appleseed/foundation/meta/tests/test_exrimagefilewriter.cpp

```diff
@@ -54,8 +54,10 @@ TEST_SUITE(Foundation_Image_EXRImageFileWriter)
         Image image(2, 2, 32, 32, 4, PixelFormatFloat);
         image.clear(Reference);
 
-        EXRImageFileWriter writer;
-        writer.write(Filename, image);
+        OIIOImageFileWriter writer{ Filename };
```

Comment:
Rename file to `test_genericimagefilewriter.cpp` (since `OIIOImageFileWriter` will be renamed to `GenericImageFileWriter`) and rename test fixture accordingly.